test/units/testsuite-58.sh

```diff
@@ -773,6 +773,58 @@ EOF
     systemd-dissect -U "$imgs/mnt"
 }
 
+test_minimize() {
+    local defs imgs output
+
+    if systemd-detect-virt --quiet --container; then
+        echo "Skipping verity test in container."
+        return
+    fi
+
+    defs="$(mktemp --directory "/tmp/test-repart.XXXXXXXXXX")"
+    imgs="$(mktemp --directory "/var/tmp/test-repart.XXXXXXXXXX")"
+    # shellcheck disable=SC2064
+    trap "rm -rf '$defs' '$imgs'" RETURN
+
+    for format in ext4 btrfs vfat; do
+        if [ ! -x "/sbin/mkfs.$format" ]; then
+            continue
+        fi
+
+        cat >"$defs/root-$format.conf" <<EOF
+[Partition]
+Type=root-${architecture}
+Format=${format}
+CopyFiles=${defs}
+Minimize=yes
+EOF
+    done
+
+    if [ -x /sbin/mksquashfs ]; then
```

Comment:
it's `/usr/bin/mksquashfs` here?

Reply:
https://packages.debian.org/sid/amd64/squashfs-tools/filelist

Reply:
Not on Fedora at least, but let's check for both I guess

Reply:
maybe instead of the actual path, which doesn't matter, use `command -v`?

Reply:
Switched to `command -v`